pulsar-broker/src/main/java/org/apache/pulsar/broker/service/TopicPoliciesService.java

```diff
@@ -80,13 +80,6 @@
      */
     void start();
 
-    /**
-     * whether the cache has been initialized.
-     * @param topicName
-     * @return
-     */
-    boolean cacheIsInitialized(TopicName topicName);
```

Comment:
The interface mark as Deprecated instead of directly delete?

Reply:
This method does not been used by the client-side, after checking the reference to this method, all of them are from the test class, I think we can delete it directly since we are not exposing the interface for users to implement their own topic policies service implementation.

Reply:
ok